src/main/frontend/src/App.tsx

```diff
@@ -0,0 +1,50 @@
+import * as React from 'react';
+import {FunctionComponent, useEffect, useState} from 'react';
+import Tab from 'react-bootstrap/Tab';
+import Tabs from 'react-bootstrap/Tabs';
+
+import RoleType from './model/RoleType';
+
+import './css/App.css';
+import 'bootstrap/dist/css/bootstrap.min.css';
+
+// @ts-ignore
+const rootUrl = rootURL;
+// @ts-ignore
+const csrfCrumb = crumb.value;
+
+
+const App: FunctionComponent = () => {
+  const [authorizationStrategy, setAuthorizationStrategy] = useState(null);
+
+  useEffect(() => {
+    (async () => {
+      const request = await fetch(`${rootUrl}/folder-auth/authorizationStrategy`, {
+        headers: {
+          'Jenkins-Crumb': csrfCrumb,
+        },
+      });
+      const data = await request.json();
+      setAuthorizationStrategy(data);
+    })().catch(err => {
+      throw new Error(`Unable to load authorization strategy: ${err}`);
+    });
+  });
```

Comment:
Perhaps you should be adding an empty [] dependency to useEffect so that fetch should happen on component mount only or add dependencies based on which you want to fetch/refetch. If no dependencies added it will refetch on every render. Although this component as it is would not re-render but if you add more states it might so better to add dependencies.

Reply:
Yes, I saw yesterday that it was causing an infinite loop. Adding `[]` fixed it. Thanks!

Reply:
Yw. And it is very interesting to see this plugin using React. Using React with Jenkins Plugin could unlock awesome powers.